docs/maven-tutorial.md

```diff
@@ -0,0 +1,222 @@
+# OpenShift + Maven + Nexus Tutorial
+
+While developing your application with Java and Maven, you'll most likely be
+building many times. In order to shorten the build times of your
+pods, Maven dependencies can be cached in a local Nexus repository.  This
+tutorial will step you through integrating a Nexus repository into your project.
+
+| Content: |
+|:----------------------------|
+|[Prerequisites](#prerequisites)|
+|[Setting up Nexus](#setting-up-nexus)|
+|[Modifying the Builder](#modifying-the-builder)|
+|[See Also](#see-also)|
+
+## Prerequisites
+
+This tutorial assumes that you'll be working with a project that is already set
+up for use with Maven.  If you're interested in using Maven with your Java
+project, we highly recommend looking at
+[their guide](https://maven.apache.org/guides/getting-started/index.html).
+
+## Setting up Nexus
+
+Download the official Nexus Docker image and use `new-app` to deploy it.
+```bash
+docker pull sonatype/nexus
+oc new-app sonatype/nexus
+```
+
+After the build has finished, run:
+```bash
+oc get svc
+```
+
+The output should resemble:
+```bash
+NAME      CLUSTER-IP       EXTERNAL-IP   PORT(S)    AGE
+nexus     172.30.170.249   <none>        8081/TCP   6s
+```
+
+Confirm that Nexus is running by navigating your browser to the new Cluster-IP
+at port 8081.  The server may take a bit of time to start, be patient!
+
+> NOTE: Nexus comes pre-configured for the Central Repository, but you may need others
+> for your application.  At this point, add any special repositories to Nexus.
+
+Now we'll create a Persistent Volume for Nexus, so that our dependencies will
+not be lost when the Pod running the server terminates.  If you do not have admin access on your cluster, ask
+your system administrator to create a Read/Write Persistent Volume for you, and
+skip to the section on creating Persistent Volume Claim.
+
+> NOTE: If you don't want persistent storage, continue to
```

Comment:
hostpath PVs aren't super useful, instead of going through this w/ the user, I suggest you instead just refer them to the official docs about how to create PVs and then say "once you have a PV defined, here's how to update your nexus deployment to use it".